gradle/distributionTesting.gradle

```diff
@@ -41,6 +41,11 @@ distributionTestTasks.all { DistributionTest task ->
 
     systemProperties['org.gradle.integtest.multiversion'] = project.hasProperty("testAllVersions") && project.testAllVersions ? 'all' : 'default'
 
+    systemProperties['org.gradle.integtest.mirrors.mavencentral'] = isCiServer ? 'http://dev12.gradle.org:8081/artifactory/repo1' : ''
```

Comment:
Instead of hardcoding these here, wouldn't it be better to receive them as environment variables?

Environment variables would make it easier to extend this strategy to other repositories such as `gradle/kotlin-dsl`.